packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -112,11 +113,17 @@ class Tooltip extends StatefulWidget {
     this.waitDuration,
     this.showDuration,
     this.child,
-  }) : assert(message != null),
-       super(key: key);
+  })  : assert(
+          tooltip != null || message != null,
+          'A non-null message or tooltip must be provided to a Tooltip Widget',
+        ),
+        super(key: key);
+
+  /// The widget to display in the tooltip.
+  final Widget? tooltip;
 
   /// The text to display in the tooltip.
-  final String message;
+  final String? message;
```

Comment:
same for this doc

Reply:
I just left the message property as it is to run the existing code as it is but can be deprecated and can introduce new semantic label property as discussed above.